app/utils/annote-marker/annote-marker.ts

```diff
@@ -1,10 +1,13 @@
 import { IconDotCircle } from "@codexteam/icons";
+
 import "./style.css";
 interface AnnoteMarkerConfig {
   placeholder?: string;
   // TODO: We may want to implement these, or not.
   onMarkerCreate?: (data: string) => void;
   onMarkerRemove?: (data: string) => void;
+  onMarkerInserted?: (data: any) => void;
+  onMarkerDeleted?: (data: any) => void;
```

Comment:
We can either use these two lines or lines 7 and 8 - either way, they are duplicates 😅